### ISSUE TITLE
Data dependancies

### ISSUE THREAD

**Issue by gitdavidhunter** [jasonhinkle/phreeze#195]
Can Phreeze generate a form to reflect table dependencies / foreign keys ? 

I have a 1-to-m between Person and Telephone. 
I want the Person form view to display the input field for Telephone which does an insert in columns in the Person table, and does the insert to the Telephone table. 
Is there a method in Phreeze to do this automatically? Or does it need to coded. 

Secondly, would be nice to have the Person form use some ajax to "add as many Telephone numbers to the Person as required".

**Comment by jasonhinkle** [comment]
I did have something like that in a much earlier non-ajax version of the generated code.  Because it's not usually possible to auto-generate anything meaningful from just the foreign keys, though, I didn't put it in the current version.

Coding it up shouldn't be that difficult, though.  The REST API for the telephone will already be there generated for you, so you just have to code the UI.

**Comment by gitdavidhunter** [comment]
Yeah ......... I see that the GetKeyMaps is being built within the DAO (PersonMap.php), what I don't understand is how to include the Telephone model in the Person Contoller in order to display the Telephone field on the Person form. 

Another example would be a 1-to-many from Person to Organization. 
Ideally, it would be nice to bring up the Person form,  and have a drop down to select the associated Organization. I don't know hwat I need to do to get this Organization dropdown into the Person form.

**Comment by jasonhinkle** [comment]
If the person/organization has a foreign key then it should show the dropdown automatically for you.  But, what it won't do is the reverse (ie show all the people on the organization page)

As far as adding the telephone as you mention, that is gonna require digging into the code a bit to understand the way the backbone.js model code works.  it looks a bit complicated at first but once you get in it, it's not too bad.

**Comment by gitdavidhunter** [comment]
OKay ........... so I don't think that is working Jason.
Perhaps there is/was something wrong with the way the tables were set up ?

Person : http://checkeredshirt.com/phreeze/phreeze-person.png
Look up table to org_loc_person : http://checkeredshirt.com/phreeze/phreeze-person-loc.png
Org : http://checkeredshirt.com/phreeze/phreeze-person-org.png

schema :
http://checkeredshirt.com/phreeze/phreeze-person.txt
